getting-started/kubernetes/hardway/install-cni-plugin.md

```diff
@@ -178,6 +178,19 @@ cat > /etc/cni/net.d/10-calico.conflist <<EOF
 EOF
 ```
 
+Step back to normal user.
+
+```bash
+exit
+```
+
+At this point Kubernetes nodes must became to `Ready` state, because Kubernetes has all the network configuration.
```

Comment:
A suggested wording adjustment here:

```
At this point Kubernetes nodes will become `Ready` because Kubernetes has a networking provider and configuration installed.
```